sdk/src/main/java/io/radar/sdk/RadarJobScheduler.kt

```diff
@@ -50,15 +51,15 @@ class RadarJobScheduler : JobService() {
             val result = jobScheduler.schedule(jobInfo)
             if (result == JobScheduler.RESULT_SUCCESS) {
                 Radar.logger.d(
-                    "Scheduling Location Job |" +
+                    "Scheduling location job |" +
+                            " jobId = $jobId;" +
                             " location = $location;" +
-                            " source = ${stringForSource(source)};" +
-                            " success = true;" +
-                            " overwrites = ${counter.getAndIncrement()}"
```

Comment:
This variable still seems helpful, since it identified the need for this change, and could be useful to identify whether or not additional changes are needed here. Can you modify it to work within this new scheduling mechanism and include it in the logs?